pkgs/games/xskat/default.nix

```diff
@@ -21,6 +21,10 @@ stdenv.mkDerivation {
   src = fetchurl {
     inherit (s) url sha256;
   };
+  preBuild = ''
+    substituteInPlace makefile --replace '/usr/bin/' "" \
```

Comment:
`Makefile`?

Reply:
Hm. The filename on darwin is actually lowercase "makefile". However replacing it with "Makefile" does work. Should I:
- Use "Makefile" and rely on the darwin filesystem being case-insensitive
- Make the preBuild phase optional stdenv.isDarwin?

Reply:
I only checked linux does it actually show up lowercase on darwin?

Reply:
`-rw-r--r--   1 lnl  staff    1679 Mar  8  2000 Makefile`

Reply:
That's weird:
```
.....
-rw-r--r--  1 nixbld1 nixbld  29K Apr 19 21:52 makefile.bak
-rw-r--r--  1 nixbld1 nixbld  80K Apr 19 21:52 makefile
drwxr-xr-x 37 nixbld1 nixbld 1.3K Apr 19 21:52 .
```

Reply:
When I put:
```
  preBuild = ''
    ls -rltah .
    exit 1
  '';
```

Reply:
preConfigure => `-rw-r--r--  1 nixbld1 nixbld   1679 Mar  8  2000 Makefile`
preBuild => `-rw-r--r--  1 nixbld1 nixbld  81101 Apr 20 00:12 makefile`

Reply:
Ok. How about `substituteInPlace ?akefile`?

Reply:
I guess, but does it actually need to run after the configurePhase? otherwise `prePatch`/`postPatch` are more appropriate IMHO.